### PR TITLE
Update UG: pe dry run bug fixes

### PR DESCRIPTION
**Pull request by ruo-x**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -170,7 +167,7 @@ Examples:
 
 Edits an existing volunteer in the volunteer list.
 
-Format: `vedit VOLUNTEER_ID [n/NAME] [p/PHONE] [e/EMAIL] [s/SKILL]…​`
+Format: `vedit VOLUNTEER_INDEX [n/NAME] [p/PHONE] [e/EMAIL] [s/SKILL]…​`
```
Comment (tiongMax) [inline docs/UserGuide.md:170]: Wait do we use index or id?

**Review by tiongMax — APPROVED** [review]
lgtm!

File: docs/UserGuide.md (hunk 1 above)
Comment (brein62) [reply docs/UserGuide.md:170]: we're using index. this is a clarity suggestion from #174

**Review by brein62 — APPROVED** [review]
LGTM!

**Pull request merged**